x-pack/legacy/plugins/alerting/server/lib/get_create_task_runner_function.ts

```diff
@@ -117,6 +117,7 @@ export function getCreateTaskRunnerFunction({
         };
 
         const alertTypeState = await alertType.executor({
+          id: alertId,
```

Comment:
Wondering if we shouldn't make this attribute `alertId` just so the semantics are clearer.

Reply:
I will update the actions plugin as well to keep them consistent. cc @pmuellr

Reply:
Fixed in: https://github.com/elastic/kibana/pull/47379/commits/e5d71a637c7514bd51984a398d04225d2dcd39a0. Actions plugin changed in: https://github.com/elastic/kibana/pull/47379/commits/b1972577c070d56541dde5bf4c790cc4a6decaba.